manual/src/tutorials/polymorphism.etex

```diff
@@ -151,9 +151,9 @@ polymorphic types. For instance, the type "'a list" is covariant in "'a":
   let f () = [];;
   let empty = f ();;
 \end{caml_example}
-Remark that the type inferred for "empty" is "'a list" and not "'_weak5 list"
-that should have occurred with the value restriction since "f ()" is a
-function application.
+Remark that the type inferred for "empty" is "'a list" and not the "'_weak5 list"
```

Comment:
Possibly "Note that" instead of "Remark that" (it reads like an instruction in speaker's notes for a talk)